.github/workflows/check-python.yml

```diff
@@ -0,0 +1,79 @@
+name: Check Python
+
+# See: https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+on:
+  push:
+    paths:
+      - ".github/workflows/check-python.yml"
+      - "**/.flake8"
+      - "**/poetry.lock"
+      - "**/pyproject.toml"
+      - "**/setup.cfg"
+      - "**/tox.ini"
+      - "**.py"
+  pull_request:
+    paths:
+      - ".github/workflows/check-python.yml"
+      - "**/.flake8"
+      - "**/poetry.lock"
+      - "**/pyproject.toml"
+      - "**/setup.cfg"
+      - "**/tox.ini"
+      - "**.py"
+  schedule:
+    # Run every Tuesday at 8 AM UTC to catch breakage caused by changes to flake8.
+    - cron: "0 8 * * TUE"
```

Comment:
I don't think this is really necessary, if we run `flake8` on every PR and never push to `main` we'll always notice breakages.

Reply:
The reason I like to use a scheduled workflow is that I have repeatedly observed breakage in CI systems occur over time due to external changes (e.g., a new rule added to flake8) and only come to light when a 3rd party contributor submits a PR and then has a bad experience from thinking that the failed CI run was caused by something they did. Running the workflow on a schedule allows that breakage to be caught by the maintainers soon after it happens.

However, in this case I don't think that's a real concern because the flake8 version is pinned by Poetry. This was adapted from a workflow that doesn't pin flake8 and I didn't consider the implications of the differences. Thanks for point that out!

I have now made the suggested change here: https://github.com/arduino/library-registry-submission-parser/compare/4dd429572daca5effec51709361a3e5ddb1c1ee2..73ec67ebea1bcfd6b7be13d66e486913c994c8c4